src/Symfony/Component/Form/FormTypeInterface.php

```diff
@@ -81,4 +81,14 @@ public function configureOptions(OptionsResolver $resolver);
      * @return string|null The name of the parent type if any, null otherwise
      */
     public function getParent();
+
+    /**
+     * Returns the prefix of the template block name for this type.
+     *
+     * The block prefixes default to the underscored short class name with
```

Comment:
Using singular "The block prefix defaults to" would fit better since this method is about exactly one prefix.